src/pages/Account.vue

```diff
@@ -1,14 +1,102 @@
 <template>
     <div>
-        <main class="container" id="account">
+        <main class="container fullwidth" id="account">
 
-            <div class="content">
+            <div class="content fullwidth">
+                <h1>Requested Payments</h1>
 
-                <h1>Hello, Market Stall Name.</h1>
+                <div class="accordion">
 
-                <p>Some more content will go here.</p>
+                    <!-- Tab header -->
+                    <div class="tab thead">
+                        <label for="tab-one">
+                            <div class="row">
+                                <div>Date</div>
+                                <div>Voucher total</div>
+                                <div>Amount</div>
+                            </div>
+                        </label>
+                    </div>
 
-                <button>Log out</button>
+                    <!-- Tab row -->
+                    <div class="tab row">
+                        <input id="tab-two" type="checkbox" name="tabs">
```

Comment:
There's going to be a lot of these tabs when we alter it to generate autmagically. when we do that this is likely to be "tab-2" . Is that going to work for you?